### PR TITLE
*: change the directory of from-aurora

### PR DESCRIPTION
**Pull request by huangxiuyan**
@dcalvin PTAL, thanks!

### CODE REVIEW

File: TOC.md [hunk 1]
```diff
@@ -41,6 +41,8 @@
     + 从 MySQL 迁移
       - [全量迁移](dev/how-to/migrate/from-mysql.md)
       - [增量复制](dev/how-to/migrate/incrementally-from-mysql.md)
+    + 从与 MySQL 兼容的数据库迁移数据
+      - [从 AWS Aurora MySQL 迁移数据](dev/how-to/migrate/from-aurora.md)
```
Comment (lilin90) [inline TOC.md]:
```suggestion
      - [从 Amazon Aurora MySQL 迁移数据](dev/how-to/migrate/from-aurora.md)
```

File: TOC.md [hunk 2]
```diff
@@ -167,7 +169,7 @@
       - [管理数据同步任务](tools/dm/manage-task.md)
       - [DM 集群操作](tools/dm/cluster-operations.md)
       + 从与 MySQL 兼容的数据库迁移数据
-        - [从 AWS Aurora MySQL 迁移数据](tools/dm/from-aurora.md)
+        - [从 AWS Aurora MySQL 迁移数据](dev/how-to/migrate/from-aurora.md)
```
Comment (lilin90) [inline TOC.md]:
```suggestion
        - [从 Amazon Aurora MySQL 迁移数据](dev/how-to/migrate/from-aurora.md)
```

File: dev/how-to/migrate/from-aurora.md [hunk 3]
```diff
@@ -1,7 +1,7 @@
 ---
 title: 从 AWS Aurora MySQL 迁移数据
 summary: 使用 DM 从 AWS Aurora MySQL 迁移数据。
-category: tools
+category: dev
```
Comment (lilin90) [inline dev/how-to/migrate/from-aurora.md]:
```suggestion
category: how-to
```

**Comment by huangxiuyan** [comment]
Resolved, and I'll preview the PRs in the file to check the display results. PTAL again. @lilin90 thanks!

**Comment by lilin90** [comment]
> * Modify all "AWS Aurora" to "Amazon Aurora"
> * Fix the Note format at the end of the from-aurora.md file (refer to that format in docs)

@huangxiuyan The second item was ignored. Please update it.

**Pull request merged**